### PR TITLE
Feature/36/GuestHomePage

### PR DESCRIPTION
**Pull request by boris-giga**
* updated eslintrc.json
* created a GuestHomePage Component
* implemented redirect to 404page if none of routes didn't match

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -2,12 +2,17 @@ import AppHeader from '~/containers/layout/AppHeader'
...
 import theme from './styles/custom-mui.styles'
+import Layout from '~/components/layout/Layout'
 
 const App = () => (
   <div className="App">
     <ThemeProvider theme={ theme }>
-      <AppHeader />
-      <AppMain />
+      <Layout>
+        { /*Router will be here*/ }
```
Comment (boris-giga) [inline src/App.js:11]: or should i place Layout inside AppMain?

Comment (maarekua) [reply src/App.js:11]: Guest Layout, yeap in App Main

File: src/containers/layout/AppHeader.js [hunk 2]
```diff
@@ -7,29 +7,31 @@ import { routes } from '~/constants/routes'
...
-        { t('common.title') }
-      </Button>
-      <Box flexGrow={ 1 } />
-      <Button color="inherit" component={ Link } to={ routes.about }>
-        { t('common.about') }
-      </Button>
-    </Toolbar>
-  </AppBar>)
+  return (
+    <AppBar position="static">
```
Comment (maarekua) [inline src/containers/layout/AppHeader.js:11]: You should create a general nav component that will be built depending on incoming props. And then create guest configuration for it.

File: src/constants/translations/en/index.js [hunk 3]
```diff
@@ -2,4 +2,4 @@ import errorPage from './errorPage.json'
 import button from './button.json'
 
 // eslint-disable-next-line import/no-anonymous-default-export
```
Comment (boris-giga) [inline src/constants/translations/en/index.js:4]: should we get rid of this?

Comment (boris-giga) [reply src/constants/translations/en/index.js:4]: same in other files

**Review by maarekua — APPROVED** [review]

**Pull request merged**